pkg/controllermanager/controller/add.go

```diff
@@ -31,5 +32,11 @@ func AddControllersToManager(mgr manager.Manager, cfg *config.ControllerManagerC
 		return fmt.Errorf("failed adding CloudProfile controller: %w", err)
 	}
 
+	if err := (&quota.Reconciler{
+		Config: cfg.Controllers.Quota,
```

Comment:
```suggestion
		Config: *cfg.Controllers.Quota,
```
You can get rid of the config pointer here, mainly because the reconciler cannot work with a nil config and the config should also be defaulted. Ref: https://github.com/gardener/gardener/commit/a173f8a62ea27ed18d02c6bb1ae69d0315e68414 (check also the commit message)